src/main-process/parse-command-line.js

```diff
@@ -134,6 +138,18 @@ module.exports = function parseCommandLine (processArgs) {
     devMode = true
   }
 
+  let projectSettings = {}
+  if (atomProject) {
+    const contents = Object.assign({}, readProjectSettingsSync(atomProject, executedFrom))
+    const config = contents.config
+    const originPath = atomProject
+    const paths = contents.paths.map((curPath) =>
+      relativizeToAtomProject(curPath, path.dirname(path.join(executedFrom, atomProject))
```

Comment:
nothing about this relativize method is Atom project-specific so the naming seems a little weird to me. also, is `executedFrom` absolute? if so, we could just do this:

```js
const pathToProjectFile = path.join(executedFrom, atomProject)
const base = path.dirname(pathToProjectFile)
const paths = contents.paths.map(curPath => path.resolve(base, curPath))
```